pkg/metrics/metrics.jsx

```diff
@@ -492,7 +518,10 @@ class CurrentMetrics extends React.Component {
         return (
             <Gallery className="current-metrics" hasGutter>
                 <Card id="current-metrics-card-cpu">
-                    <CardTitle>{ _("CPU") }</CardTitle>
+                    <CardHeader>
+                        { isNaN(this.cpuTemperature) ? null : <CardActions>{ this.cpuTemperature + " °C"}</CardActions> }
```

Comment:
I remember rather angry UK users who (rightfully to a large degree) complain about US-isms like "broken date format" in our UI. Supporting en-GB separately is not a drive-by fix, though. Possibly we should just show both Celsius and Fahrenheit for the time being?

Also, please don't combine strings that way. You need to use `cockpit.format(_("$0 °C"), temp)` so that this can be localized. This is even more important if you make the string more complex.